ckanext/bcgov/templates/package/snippets/map_view.html

```diff
@@ -22,8 +22,9 @@
       {% else %}
       	{% set map_service_url = pkg.preview_map_service_url %}
         	{% if map_service_url %}
-      		{% set wmsservice_url = map_service_url + "&wmslayers=" + pkg.layer_name + "&ll=" + pkg.preview_latitude + "," + pkg.preview_longitude + "&z=" + pkg.preview_zoom_level %}
-      		{% set resource_url = "//apps.gov.bc.ca/pub/dmf-viewer/?siteid=7535188336326689232&maponly&wmsservices=" + wmsservice_url %}
+          {% set wmsservice_url = map_service_url + "&wmslayers=" + pkg.layer_name + "&ll=" + pkg.preview_latitude + "," + pkg.preview_longitude + "&z=" + pkg.preview_zoom_level %}
```

Comment:
Do we need both lines of code that set the `wmsservice_url` variable, or just the new one at line 26?

Reply:
I just merged the two lines

Reply:
Sorry, I don't understand.  Did you just make a new code change?

Reply:
No the resource line is still in the code, it just isn't showing in this for some reason

Reply:
Reviewed with Brandon.  Line 25 maintained as is in case we need to revert.